target/scripts/helpers/utils.sh

```diff
@@ -34,3 +34,28 @@ function _chown_var_mail_if_necessary
     chown -R 5000:5000 /var/mail || return 1
   fi
 }
+
+function _require_n_parameters_or_print_usage
+{
+  local COUNT FAIL=0
+  COUNT=${1}
+  shift
+
+  # One or more parameters?
+  if [[ ${COUNT} -eq 1 ]]; then
+    # one parameter
+    if [[ ${1:-} == 'help' ]] || [[ -z ${1:-} ]]
+    then
+      FAIL=1
+    fi
+  else
+    # n parameters
+    [[ ${#} -lt ${COUNT} ]] && FAIL=1
+  fi
+
+  if [[ ${FAIL} -eq 1 ]]
+  then
+    __usage
+    exit 0
+  fi
```

Comment:
Thanks for doing this :)

Would this be simpler? Or does it not cover something well?

```suggestion
  if [[ ${1:-} == 'help' ]] || [[ -z ${1:-} ]] || [[ ${#} -lt ${COUNT} ]]
    then
    __usage
    exit 0
  fi
```

Reply:
I think so 😆. I also had another idea that I want to distinguish between 'help' is given and everything else: `exit 0` / `exit 1`.

Reply:
Does this work? :sweat_smile:

```bash
  if [[ ${1:-} == 'help' ]] || [[ -z ${1:-} ]] || [[ ${#} -lt ${COUNT} ]]
    then
    __usage

    [[ ${1:-} == 'help' ]] && exit 1
    exit 0
  fi
```

I'm not sure if the `-z` condition is doing anything for us though? Doesn't the `-lt` also cover that?

Reply:
Love this 🚀